terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -239,10 +239,10 @@
 # For instructions on finding the latest CIS-hardened AMI, see
 # OPERATOR.rst#upgrading-linux-ami
 #
-# CIS Amazon Linux 2 Kernel 4.14 Benchmark v2.0.0.29 - Level 1-4c096026-c6b0-440c-bd2f-6d34904e4fc6
+# CIS Amazon Linux 2 Kernel 4.14 Benchmark - Level 1 - v03 -4c096026-c6b0-440c-bd2f-6d34904e4fc6
```

Comment:
We can't upgrade to this.

https://github.com/DataBiosphere/azul/issues/6082

Please add a FIXME here so that we don't mistakenly attempt this upgrade again until that issue has been resolved.